common/models/express-usage-record.json

```diff
@@ -15,6 +15,33 @@
       "type": "date",
       "required": true
     },
+    "requestMethod": {
+      "type": "string"
+    },
+    "requestUrl": {
+      "type": "string"
+    },
+    "clientAddress": {
+      "type": "string"
+    },
+    "clientUsername": {
+      "type": "string"
+    },
+    "clientId": {
+      "type": "string"
+    },
+    "clientDetail": {
+      "type": "object"
+    },
+    "lbModelName": {
+      "type": "string"
+    },
+    "lbInstanceId": {
+      "type": "string"
+    },
+    "lbRemoteMethod": {
+      "type": "string"
+    },
```

Comment:
For now this model is stored in memory but we may decide to store this in a sql db soon. In that case, I wont be able to query/order on this field unless it is at the top level.